Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/InstallDiffCommand.cs

```diff
@@ -134,7 +135,7 @@ private IUnarchiver CreateUnrachiver(string destinationDir)
                 case "zip":
                     return new ZipUnarchiver(_packagePath, destinationDir, _packagePassword);
```

Comment:
ZipUnarchiver should also have support for suffixes.

Reply:
Turns out it's not possible. Ionic.Zip doesn't support suffixes for unarchiving files.